user_funcs.py

```diff
@@ -116,3 +117,246 @@ def login(conn) -> tuple[str, int]:
                 return results[0]
 
             print("Username or password incorrect!")
+#THis whole function does work correctly
+def create_collection(conn, user_ID) -> None:
```

Comment:
NIT: For snake case, all variables should be completely lower case
```suggestion
def create_collection(conn, user_id) -> None:
```